rest/model/distro.go

```diff
@@ -9,12 +9,24 @@ import (
 )
 
 // APIDistro is the model to be returned by the API whenever distros are fetched.
-// EVG-1717 will implement the remainder of the distro model.
 type APIDistro struct {
-	Name             APIString `json:"name"`
-	UserSpawnAllowed bool      `json:"user_spawn_allowed"`
-	Provider         APIString `json:"provider"`
-	ImageID          APIString `json:"image_id,omitempty"`
+	Name             APIString              `json:"name"`
+	UserSpawnAllowed bool                   `json:"user_spawn_allowed"`
+	Provider         APIString              `json:"provider"`
+	ProviderSettings map[string]interface{} `json:"settings,omitempty"`
+	ImageID          APIString              `json:"image_id,omitempty"`
+	Arch             APIString              `json:"arch,omitempty"`
+	WorkDir          APIString              `json:"work_dir,omitempty"`
+	PoolSize         int                    `json:"pool_size,omitempty"`
+	SetupAsSudo      bool                   `json:"setup_as_sudo,omitempty"`
+	Setup            APIString              `json:"setup,omitempty"`
+	Teardown         APIString              `json:"teardown,omitempty"`
+	User             APIString              `json:"user,omitempty"`
+	SSHKey           APIString              `json:"ssh_key,omitempty"`
+	SSHOptions       []string               `json:"ssh_options,omitempty"`
+	Expansions       map[string]string      `json:"expansions,omitempty"`
+	Disabled         bool                   `json:"disabled,omitempty"`
+	ContainerPool    APIString              `json:"container_pool,omitempty"`
```

Comment:
I think we want to have omit empty only on fields that we're filtering out? we could also omit-empty nothing, and I think it'd be fine.